src/main/java/com/mycompany/thevoid/Character.java

```diff
@@ -59,39 +58,12 @@ public void setMods() {
         int currentConMod = StatsMods[2];
 
         for (int i = 0; i < 6; i++) {
-            if (Stats[i] == 30) {
+            if(Stats[i] > 30) {
```

Comment:
Unfortunately this approach will result in a wrong output, I will upload a picture of what results are needed depends on Stats[i]

![stats mods dnd](https://user-images.githubusercontent.com/63417518/190337853-6516eac8-9219-4fdb-8701-5d29cb201bf4.png)

Reply:
Oh sorry, actually I missed one more line, will be updating it now.
We can remove the **if( Stats[i] > 30)** condition if the score is not going beyond 30, and then it will result in the same as shown in the above image.

Reply:
Hi @Jihanger-997, will the **Stats[i]** ever > 30 in any of the conditions? 
if it will then I can add a check for it.

Reply:
@Parneet-Raghuvanshi yes, they can be, but only for monsters (Enemy)

Reply:
Ok